### ISSUE TITLE
[1.3-rc.3] browser-policy disallowInline missing NODE_ENV, causing Meteor undefined on client.

### ISSUE THREAD

**Issue by MGovier** [meteor/meteor#6539]
Disallowing inline scripts using [browser-policy](https://github.com/meteor/meteor/tree/devel/packages/browser-policy) is causing errors on client initialisation in 1.3-rc.3:
```
Uncaught TypeError: Cannot read property 'NODE_ENV' of undefined(anonymous function) @ meteor.js?hash=b079ca6…:59
Uncaught TypeError: Cannot read property 'Meteor' of undefined
```
It looks like the separate `meteor-runtime-config.js` that is created to avoid the normally inlined runtime config is missing the NODE_ENV definition.

#### Repro
```
meteor create --examples todos
cd todos
meteor add browser-policy
```
Add `BrowserPolicy.content.disallowInlineScripts();` to `server/bootstrap.js` startup function. 
This works fine on 1.2.1, but after:
```
meteor update --release 1.3-rc.3
meteor
```
the above error occurs. 

Quite a niche one - cheers for all the great work on the new release!

**Comment by tmeasday** [comment]
@benjamn I think we need to either perma-extend `__meteor_runtime_config__` with `meteorEnv`, or do here: https://github.com/meteor/meteor/blob/release-1.3/packages/webapp/webapp_server.js#L282-L286

what we do here:
https://github.com/meteor/meteor/blob/release-1.3/packages/webapp/webapp_server.js#L361-L362

**Comment by benjamn** [comment]
Fixed in [`1.3-rc.4`](https://github.com/meteor/meteor/issues/6266#issuecomment-198997482).
